hw/dv/data/dsim/dsim.hjson

```diff
@@ -0,0 +1,122 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "{job_prefix} dsim"
+  build_ex:   "dsim"
```

Comment:
This appears to be unused and can be removed. Have the `run_cmd` set to `"{job_prefix} dsim"` instead.